network_monitor.py

```
@@ -0,0 +1,299 @@
+import os
+import sys
+import time
+import threading
+import random
+
+
+import os
```

Ruff (0.12.2)

Redefinition of unused `os` from line 1

Remove definition: `os`

(F811)